.github/workflows/workflow.yml

```diff
@@ -1,36 +1,55 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
-name: 'CodeQL'
+name: Workflow
 
 on:
   push:
     branches: ['develop', main]
   pull_request:
-    # The branches below must be a subset of the branches above
-    branches: ['develop']
+    branches:
+      - '**'
   schedule:
     - cron: '35 9 * * 6'
 
 jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'schedule'
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
Why 14 as 14 is quite older now go for lts

Reply:
Updating to active [LTS](https://nodesource.com/blog/Node.js-v22-Long-Term-Support-LTS)

Reply:
Increasing the version, increases the time from 1 to 5 minutes. Increasing time. Should we go with this?